pipelinewise/cli/pipelinewise.py

```diff
@@ -1985,6 +2007,38 @@ def _clean_tables_from_bookmarks_in_state_file(state_file_to_clean: str, tables:
         except json.JSONDecodeError:
             pass
 
+    @staticmethod
+    def _get_fixed_name_of_table(stream_id):
+        return stream_id.replace('-', '.', 1)
+
+    def _get_sync_tables_setting_from_selection_file(self, tables):
+        selection = utils.load_json(self.tap['files']['selection'])
+        selection = selection.get('selection')
+        all_tables = {'full_sync': [], 'partial_sync': {}}
+        if selection:
+            for table in selection:
+                table_name = self._get_fixed_name_of_table(table['tap_stream_id'])
+                if tables is None or table_name in tables:
+                    if table.get('sync_start_from'):
+                        all_tables['partial_sync'][table_name] = table['sync_start_from']
+                    else:
+                        all_tables['full_sync'].append(table_name)
+            return all_tables
+
+    def _do_not_run_if_another_instance_is_running(self, sync_method):
```

Comment:
We stopped relying on log files to figure out if a tap is running, this has led to incidents before.
https://transferwise.atlassian.net/browse/AP-1104

Reply:
- [ ] Remove log file checking for running